sherpa/astro/tests/test_astro_data_notebook.py

```diff
@@ -282,3 +321,48 @@ def test_img_real_filtered(coord, make_data_path,
     assert '<div class="dataval">[ 4096.5  4096.5]</div>' in r
     assert '<div class="dataval">[-0.000137  0.000137]</div>' in r
     assert '<div class="dataval">destreak - CAT3.0.2</div>' in r
+
+
+def test_low_level_calc_erange_constant():
+    """Easiest just to call this directly rather than fake up data"""
```

Comment:
No need for excuses. That's what unit tests do: They test individual units (Pun intended, but I admit it does not work well)!